### PR TITLE
link to new function names

### PR DESCRIPTION
**Pull request by d-morrison**
adding hyperlinks from `llik` documentation to `log_likelihood` documentation, and similarly for `fdev`->`f_dev`

### CODE REVIEW

**Review by kristinawlai — APPROVED** [review]

**Comment by kristinawlai** [comment]
@d-morrison This looked good to me so I approved, but then when I merged and looked at the documentation in my RStudio the links don't appear (only the plain function names). Would updating it to the code below (using `@description` ) work for you? This seems to move this text into a Details section below Usage, so it's less prominent, but the link works. 

```
#' Calculate log-likelihood
#'
#' `r lifecycle::badge("deprecated")`
#'
#' @description `llik()` was renamed to \code{\link{log_likelihood}} to create a more
#' consistent API.
#'
#' @keywords internal
#' @export
#'
llik <- function(
    lambda,
    pop_data,
    antigen_isos,
    curve_params,
    noise_params,
    verbose = FALSE,
    ...)
{
  lifecycle::deprecate_warn("1.0.0", "llik()", "log_likelihood()")
  log_likelihood(lambda, pop_data, antigen_isos, curve_params, noise_params, verbose = FALSE)


}
```

**Comment by d-morrison** [comment]
Good catch!

If we put the ***@***.***` tag on the line before `r lifecycle::badge("deprecated")`, that should fix the issue with the text appearing in Details:

[cid:e695441c-2bc1-436c-b233-88efacb3c8c0]

[cid:4c4bc31e-fe00-428f-8c97-485da0ca47f6]

Here's a patch PR with the added @description tag: https://github.com/UCD-SERG/serocalculator/pull/184

By the way, to quickly switch to a PR branch, I recently started using the usethis::pr_fetch() function; just give it the PR number, for example`usethis::pr_fetch(184)` to get the PR above. Then you can test out a PR on your local computer before merging it. It's a bit easier than finding the branch name corresponding to the PR.

Sometimes I have to restart R or RStudio to get documentation changes to appear, and I've found it's more reliable to reinstall instead of taking the load_all() shortcut. Sometimes the hyperlinks haven't worked with `load_all()`, although they are working today (I recently updated the `roxygen2` package, so maybe that fixed it). If you use load_all(), you'll get a "(preview)" tag in the upper right corner, which can remind you that you're not necessarily seeing fully formatted documentation:

[cid:2fdd3540-232a-419e-9135-8872892ba1c9]

Hope that helps!

Best,
Ezra

________________________________

**Pull request merged**
